src/pages/search/index.js

```diff
@@ -446,23 +451,17 @@ export class Search extends Component {
     if (e === 'recent sales') {
       let tokens = await fetchSales(this.state.offset)
       tokens = tokens.map((e) => e.token)
-      tokens = tokens.filter((e) => !arr.includes(e.creator_id))
+      tokens = tokens.filter(filterBan)
       this.setState({
-        feed: _.uniqBy(
-          _.uniqBy([...this.state.feed, ...tokens], 'id'),
-          'creator_id'
-        ),
+        feed: _.uniqBy([...this.state.feed, ...tokens], 'id'),
       })
     }
 
     if (e === 'new OBJKTs') {
       let tokens = await fetchFeed(this.state.lastId, this.state.offset)
-      tokens = tokens.filter((e) => !arr.includes(e.creator_id))
```

Comment:
I would actually add two conditions for the filter in the new feed.

We could have a dict to track how many tokens have been displayed in the feed for a given artist_address. Then in each batch we do the following:
- if the artist counter is <= 10, then we display the token and increase the counter.
- if the artist counter is > 10, then we only display the first token of that artist in that batch and filter the rest.

In this way if a user mints 10 tokens in the same batch, the tokens will still be shown.

If a user mints 200 tokens, the first 10 tokens will be shown, and after that we will see only one token per batch (1 every 45). So, if one goes very down in the feed, we will still be able to find old mints for normal users, because we have the 1 per batch option.